### ISSUE TITLE
Digest and Email notifications

### ISSUE THREAD

**Issue by ChrisChinchilla** [drewkeller/redmine_digest#27]
Just wondering, does the digest respect Users Email notifications setting? i.e. if they're set to 'no events', will the digest still be delivered?

**Comment by drewkeller** [comment]
No, the digest is independent from the email notifications setting.

So.... (assuming the administrator has the digest set to once per day 
and the user enables his digest)...
- If the user sets email notifications to 'no events', he will receive 
  ONLY ONE email per day with all events . The boss would probably want it 
  set this way.
- If the user sets email notifications to 'any event on all my 
  projects', he will receive an immediate email for each event when it 
  occurs AND a digest email once per day containing all events. The 
  project/web administrator may want it set this way.

An improvement would be to allow the user to set the period for his own 
digest email. However, the logic of the plugin would need to be 
rewritten to send out individual emails. Patches are welcome, as always.

**Comment by ChrisChinchilla** [comment]
Great! That's what I wanted, the reason for installing the plugin was because people were complaining of getting too many notification emails, but I still want them to be reminded of things that need doing!

Great work, thanks :)
